index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="navbar">
+
+        <ul class="nav-list">
+          <li>
+            <a href="#welcome-section">About</a>
+          </li>
+          <li>
+            <a href="#projects">Projects</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
+
+        </ul>
+    </div>
+
+    <section id="welcome-section">
+        <h1>Hello World, we are software developers!</h1>
+        <p>from Microverse</p>
+    </section>
+
+
+
+    <section id="projects">
+
+      <h1 class="projects-title">Some of our projects</h1>
+      <ul class="projects-grid">
+        <li class="project-tile"><a href=""><img class="project-img" alt="img1" src="https://picsum.photos/200"><p class="project-title">Project</p></a></li>
+        <li class="project-tile"><a href=""><img class="project-img" alt="img1" src="https://picsum.photos/200"><p class="project-title">Project</p></a></li>
+        <li class="project-tile"><a href=""><img class="project-img" alt="img1" src="https://picsum.photos/200"><p class="project-title">Project</p></a></li>
+        <li class="project-tile"><a href=""><img class="project-img" alt="img1" src="https://picsum.photos/200"><p class="project-title">Project</p></a></li>
+        <li class="project-tile"><a href=""><img class="project-img" alt="img1" src="https://picsum.photos/200"><p class="project-title">Project</p></a></li>
+        <li class="project-tile"><a href=""><img class="project-img" alt="img1" src="https://picsum.photos/200"><p class="project-title">Project</p></a></li>
+      </ul>
+    </section>
+
+    <section id="contact">
```

Comment:
- [ ] [OPTIONAL] For best practice please avoid id's in order to make your code consistency because classes are reusable